### PR TITLE
User management and notifications/TEAM1-(22-28)

### PR DESCRIPTION
**Pull request by david-khachatryan**
Auth forms, password validation, info pages after success, redirection to "/login"

### CODE REVIEW

**Review by MMihran — APPROVED** [review]

**Review by MMihran — APPROVED** [review]

File: src/src_V2/components/authInfoWrapper/index.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+export default function AuthInfoWrapper({ ...props }) {
+  return (
+    <div className={styles.wrapper}>
+      <img className={styles.logo} src={logo} alt='' />
+
+      <div className={styles.content}>{props.children}</div>
+    </div>
+  );
+}
```
Comment (igortokhunts) [inline src/src_V2/components/authInfoWrapper/index.jsx]: @david-khachatryan jan why we need this in the global components folder i think it is auth related not global ones and need to be placed in auth module.

File: src/src_V2/components/authWrapper/authPageWrapper.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            </Typography>
+          )}
+        </div>
+        <div className={styles.formWrapper}>{props.children}</div>
+      </div>
+    </div>
+  );
+};
+
+export default authPageWrapper;
```
Comment (igortokhunts) [inline src/src_V2/components/authWrapper/authPageWrapper.jsx]: The same comment for folder structure as reviouse.

File: src/src_V2/components/congratulations/index.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import styles from './styles.module.css';
+import { CircularProgress, Typography } from '@mui/material';
+import { useEffect } from 'react';
+
+export default function Congratulations({ urlToRedirect, timeout }) {
+  const params = new URLSearchParams(location.search);
+  let navigate = useNavigate();
+
+  useEffect(() => {
+    if (!params.get('token')) {
```
Comment (igortokhunts) [inline src/src_V2/components/congratulations/index.jsx]: @david-khachatryan jan why we need get params from URLSearchParams i think we can get it from react-router-dom, it has useParams hook, and also for the setTimeout what is the reason of this api usage?

File: src/src_V2/components/formUI/PasswordField/index.jsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
+import { TextField } from '@mui/material';
+import { useState } from 'react';
+import InputAdornment from '@mui/material/InputAdornment';
+import IconButton from '@mui/material/IconButton';
+import Visibility from '@mui/icons-material/Visibility';
+import VisibilityOff from '@mui/icons-material/VisibilityOff';
+import { Field, useFormikContext } from 'formik';
```
Comment (igortokhunts) [inline src/src_V2/components/formUI/PasswordField/index.jsx]: @david-khachatryan jan please also pay attention the imports style, with this ordering
1. library imports
2. local component imports
3. constat imports

File: src/src_V2/components/verifyEmail/index.jsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+          Please check your <b>{props.email}</b> email inbox for a verification email.
+        </Typography>
+        <Typography variant='subtitle1'>
+          Didn&apos;t receive an email?{' '}
+          <Link
+            className={clsx({
+              [styles.link]: true,
+              [styles.disabledLink]: secondsRemain > 0,
+            })}
+            onClick={() => {
```
Comment (igortokhunts) [inline src/src_V2/components/verifyEmail/index.jsx]: i think we don't need aditional function wrapper here

File: src/src_V2/modules/auth/login/Login.jsx [hunk 6]
```diff
@@ -0,0 +1,62 @@
+import { Link, Typography } from '@mui/material';
+import { Link as RouterLink } from 'react-router-dom';
+import AuthPageWrapper from '../../../components/authWrapper/authPageWrapper';
```
Comment (igortokhunts) [inline src/src_V2/modules/auth/login/Login.jsx]: the same comment for importing styles as above.

File: src/src_V2/modules/auth/signUp/SignUp.module.css [hunk 7]
```diff
@@ -0,0 +1,3 @@
+.fieldsWrapper {
+  margin-bottom: 16px;
+}
```
Comment (igortokhunts) [inline src/src_V2/modules/auth/signUp/SignUp.module.css]: Please run prettier on for this kind of cases(new line and etc.)

File: src/src_V2/components/authInfoWrapper/index.jsx [hunk 8]
```diff
@@ -0,0 +1,12 @@
+import styles from './styles.module.css';
+import logo from '../../assets/Travely.png';
+
+export default function AuthInfoWrapper({ ...props }) {
+  return (
+    <div className={styles.wrapper}>
```
Comment (TigranYengiba04) [inline src/src_V2/components/authInfoWrapper/index.jsx]: pls use Box

File: src/src_V2/components/authInfoWrapper/index.jsx [hunk 9]
```diff
@@ -0,0 +1,12 @@
+import styles from './styles.module.css';
+import logo from '../../assets/Travely.png';
+
+export default function AuthInfoWrapper({ ...props }) {
```
Comment (TigranYengiba04) [inline src/src_V2/components/authInfoWrapper/index.jsx]: pls destructure { children } and use it in line 9

File: src/src_V2/components/authWrapper/authPageWrapper.jsx [hunk 10]
```diff
@@ -0,0 +1,32 @@
+import styles from './authPageWrapper.module.css';
+import logo from '../../assets/Travely.png';
+import loginIllustration from '../../assets/login_illustration.svg';
+import { Typography } from '@mui/material';
+
+const authPageWrapper = (props) => {
```
Comment (TigranYengiba04) [inline src/src_V2/components/authWrapper/authPageWrapper.jsx]: pls use destructuring, and use Mui components to render your UI.

File: src/src_V2/components/formUI/Button/index.jsx [hunk 11]
```diff
@@ -0,0 +1,26 @@
+import { Button } from '@mui/material';
+import { useFormikContext } from 'formik';
+
+const ButtonWrapper = ({ children, ...otherProps }) => {
+  const context = useFormikContext();
+  const { submitForm } = context;
+
+  const handleSubmit = () => {
```
Comment (TigranYengiba04) [inline src/src_V2/components/formUI/Button/index.jsx]: pls remove handleSubmit and use submitForm

**Review by TigranYengiba04 — CHANGES_REQUESTED** [review]

File: src/src_V2/modules/auth/registerAgency/RegisterAgency.module.css [hunk 12]
```diff
@@ -0,0 +1,3 @@
+.fieldsWrapper {
+    margin-bottom: 16px;
+}
```
Comment (igortokhunts) [inline src/src_V2/modules/auth/registerAgency/RegisterAgency.module.css]: newline

File: src/src_V2/components/authInfoWrapper/index.jsx (hunk 8 above)
Comment (david-khachatryan) [reply src/src_V2/components/authInfoWrapper/index.jsx]: Although we use MUI, but without using the functionality of the Box I think there is no need to replace the div with the Box. After all it's a component with it's own logic.

File: src/src_V2/components/authInfoWrapper/index.jsx (hunk 9 above)
Comment (david-khachatryan) [reply src/src_V2/components/authInfoWrapper/index.jsx]: Thanks.

File: src/src_V2/components/authWrapper/authPageWrapper.jsx (hunk 10 above)
Comment (david-khachatryan) [reply src/src_V2/components/authWrapper/authPageWrapper.jsx]: Thanks.

File: src/src_V2/components/formUI/Button/index.jsx (hunk 11 above)
Comment (david-khachatryan) [reply src/src_V2/components/formUI/Button/index.jsx]: In the initial state that didn't work, thanks for noticing.

**Review by igortokhunts — APPROVED** [review]

**Review by sasuntitan — APPROVED** [review]

**Pull request merged**